lib/services/xmpp_im.rb

```diff
@@ -15,9 +15,11 @@ class Service::XmppIm < XmppHelper
   default_events :push, :commit_comment, :issue_comment,
     :issues, :pull_request, :pull_request_review_comment,
     :gollum
- 
+
   def send_messages(messages)
     messages = Array(messages)
+    messages << Service::DEPRECATION_NOTE
```

Comment:
🤔 I'm not sure if we should be [similarly](https://github.com/github/github-services/pull/1246/files#r197970042) breaking this up into separate lines, or will XMPP accept multi-line text.

@gjtorikian any idea?

Reply:
[This issue](https://github.com/xmpp4r/xmpp4r/issues/27#issuecomment-42639318) suggests that multiple lines are fine. 🤷🏻‍♂️